staticd/static_nb_config.c

```diff
@@ -461,6 +468,10 @@ int routing_control_plane_protocols_control_plane_protocol_staticd_route_list_cr
 				yang_dnode_get_string(args->dnode, "./prefix"));
 			return NB_ERR;
 		}
+		if (vrf->vrf_id == VRF_UNKNOWN)
+			zlog_warn(
+				"Static Route to %s not installed currently because dependent config not fully available",
```

Comment:
Now that #6920 was merged, logging to `args->errmsg` should make the NB return this warning back to the CLI user as expected.